include/constants/global.h

```diff
@@ -64,6 +64,8 @@
 #define SAVED_TRENDS_COUNT 5
 #define PYRAMID_BAG_ITEMS_COUNT 10
 
+#include "battle_config.h"
```

Comment:
```suggestion
#include "constants/battle_config.h"
```
There isn't a `include/battle_config.h` file in any of the 3 branches.
Also, this should be put at the top of the file, below `#define GUARD_CONSTANTS_GLOBAL_H` imo in order to remain consistent with the huge majority of the files in the project, kinda like this:
```
#define GUARD_CONSTANTS_GLOBAL_H

#include "constants/battle_config.h"

// Invalid Versions show as "----------" in Gen 4 and Gen 5's summary screen.
```